tests/modules/bamtools/convert/test.yml

```diff
@@ -0,0 +1,92 @@
+- name: bamtools convert test_bamtools_convert_ext_error
+  command: nextflow run tests/modules/bamtools/convert -entry test_bamtools_convert_ext_error -c tests/config/nextflow.config
+  tags:
+    - bamtools
+    - bamtools/convert
+    - ext_error
+  exit_code: 1
+
+- name: bamtools convert test_bamtools_convert_noext_error
+  command: nextflow run tests/modules/bamtools/convert -entry test_bamtools_convert_noext_error -c tests/config/nextflow.config
+  tags:
+    - bamtools
+    - bamtools/convert
+    - noext_error
```

Comment:
Annoyingly - you can not put in custom tags - I have also wanted to do this before

Reply:
😄 No problems, I removed those. 
The tests passed locally. Let's on github.